go.mod

```diff
@@ -6,13 +6,12 @@ replace k8s.io/client-go => k8s.io/client-go v0.20.2
 
 require (
 	github.com/go-logr/logr v0.3.0
-	github.com/rancher/eks-operator v1.0.6-rc1
-	github.com/rancher/fleet/pkg/apis v0.0.0-20210203165831-44af1553b47e
+	github.com/rancher/fleet/pkg/apis v0.0.0-20210225010648-40ee92df4aea
```

Comment:
qq: eks-operator got removed, is this expected? I still see it in master latest commit

Reply:
I had to run `go mod tidy` after I fixed the merge conflicts and it was removed.

Reply:
Does master branch of operator already avoid the cluster-overwrite problem ?

Reply:
Darren changed this part of it, so I would guess that it wouldn't have this problem anymore. Regardless, Daishan's PR should also fix the overwrite problem: https://github.com/rancher/rancher-operator/pull/34